presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestApproximateCountDistinctTinyint.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.aggregation;
+
+import com.facebook.presto.metadata.Signature;
+import com.facebook.presto.spi.type.Type;
+
+import java.util.concurrent.ThreadLocalRandom;
+
+import static com.facebook.presto.metadata.FunctionKind.AGGREGATE;
+import static com.facebook.presto.spi.type.BigintType.BIGINT;
+import static com.facebook.presto.spi.type.DoubleType.DOUBLE;
+import static com.facebook.presto.spi.type.TinyintType.TINYINT;
+
+public class TestApproximateCountDistinctTinyint
```

Comment:
I agree that having this kind of tests for every single type is an overkill. But just for curiosity, why did you chose those types (`LONG DECIMAL`, `INTEGER`, `IPADDRESS`, `SMALLINT`, `TINYINT`) over the others?

Reply:
I chose these types because their underlying representations are different. Some of them may only have different length, but I want to test if the standard error holds for different value space. 

Types I skipped have the same internal representation as `BIGINT`: 
`SHORT DECIMAL`, `TIMESTAMP`, `DATE`, `TIMESTAMP WITH TIMEZONE`, `TIME`, `TIME WITH TIMEZONE`, `INTERVAl YEAR TO MONTH`, `INTERVAL DAY TO SECOND`.